.circleci/config.yml

```diff
@@ -124,6 +123,18 @@ jobs:
             if [[ -z "$version" ]] ; then version=${CIRCLE_SHA1:0:7}; fi
             aws s3 sync ~/artifacts s3://honeycomb-builds/honeycombio/refinery/$version/
 
+  publish_docker:
+    executor: linuxgo
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: go install github.com/google/ko@latest
+      - run:
+          name: "Publish multi-arch docker image"
+          command: |
+            echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin;
+            VERSION=${CIRCLE_TAG} KO_DOCKER_REPO=honeycombio build/build_docker.sh
```

Comment:
Recommendation: also perform a local ko build during the test or build job so we can see build errors earlier (on PRs and not just on taggings).